.changeset/three-pants-punch.md

```diff
@@ -0,0 +1,9 @@
+---
+'@eth-optimism/contracts-bedrock': minor
```

Comment:
I don't think we need to do another minor release but not super opinionated. This package will be deprecated and we will do a major release of `contracts` once we upgrade to bedrock

Reply:
changed to patch